audio-worklet/design-pattern/README.md

```diff
@@ -0,0 +1,24 @@
+# WebAudio + WebAssembly
+
+A collection of examples demonstrate the integration of Web Audio API and
+WebAssembly. Performing custom audio processing with Web Audio API can be done
+in two ways:
+[AudioWorklet](https://developers.google.com/web/updates/2017/12/audio-worklet)
+and ScriptProcessorNode (deprecated). Both examples can be found here.
+
+  - [Live example pages]()
+
+
+## Development
+
+To build the source (C/C++) in the examples,
+[Emscripten](http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html)
+MUST be installed with its prerequisites. Each example has its own Makefile, so
```

Comment:
I think this reads better if you have Emscripten link to the main emscripten site, and have "MUST be installed" (or some substring) link to the installation instructions.